### ISSUE TITLE
importlib issue

### ISSUE THREAD

**Issue by anvoynov** [NVlabs/stylegan2-ada-pytorch#239]
**Describe the bug**
In certain environments `training_stats` module has different ids inside the loss class and `training_loop`. This makes discriminator signs statistics unreachable from `training_loop`.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.5
 - PyTorch version 1.10.1+cu111
 - CUDA toolkit version 11.4
 - NVIDIA driver version 470.103.01
 - GPU A100
 - Python 3.8.8

**Additional context**
I believe the issue appears from the way a loss class is created:
https://github.com/NVlabs/stylegan2-ada-pytorch/blob/6f160b3d22b8b178ebe533a50d4d5e63aedba21d/training/training_loop.py#L195

as internally it calls `importlib`, it may initiate an independent imports that leads to two `training_stats` instances:
https://github.com/NVlabs/stylegan2-ada-pytorch/blob/6f160b3d22b8b178ebe533a50d4d5e63aedba21d/dnnlib/util.py#L237
